src/response.rs

```diff
@@ -436,6 +436,16 @@ where
 
         Ok(())
     }
+
+    /// Retrieves the current value of the `Response` status code
+    pub fn get_status_code(&self) -> StatusCode {
```

Comment:
I think the "get_" prefix is not used anywhere else in the public API:s of this codebase. `status_code` and `data_length` would do fine

Reply:
Sure thing! I will replace those with the names suggested!